client/my-sites/plans/jetpack-plans/product-store/product-simple-card/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { ItemPrice } from '../item-price';
+import { ProductSimpleCardProps } from '../types';
+import getProductIcon from '../utils/get-product-icon';
+
+import './style.scss';
+
+const ProductSimpleCard: React.FC< ProductSimpleCardProps > = ( {
+	item,
+	isOwned,
+	siteId,
+	onClickMore,
+	onClickPurchase,
+	checkoutURL,
+} ) => {
+	const translate = useTranslate();
+
+	const productIcon = useMemo(
+		() => getProductIcon( { productSlug: item.productSlug } ),
+		[ item ]
+	);
```

Comment:
No need to memoize this IMHO.

Reply:
You are right 👍 , getProductIcon isn't really an expensive calculation so its better not to useMemo here.